### PR TITLE
Add underlines for links in paragraphs for Paultons redesign

### PR DESCRIPTION
**Pull request by jlsorak**
#### What does this PR do? (please provide any background)
Adds and underline to links that are within paragraph text distinguish them from the surrounding text.  

#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### Screenshots / Screencast

Before: 
screen shot 2018-01-04 at 11 59 58

After: 

#### What gif best describes how you feel about this work?

- [ ] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

File: brands/paultons/src/less/type.less [hunk 1]
```diff
@@ -1 +1,9 @@
 @import '../../../brand-globals/src/less/type';
+
+//
+// Typography
+// --------------------------------------------------
+
+p > a { 
+  text-decoration:underline;
+}
```
Comment (putvande) [inline brands/paultons/src/less/type.less]: Is this not going to give unwanted effects? Is it just those boxes that need an underline for links, or does every link on the website that is within a `<p>` tag need an underline?

Comment (jask84) [reply brands/paultons/src/less/type.less]: really pedantic but generally more accepted to have a space between property and value? `text-decoration: underline`.

However, looking at other less files in this directory, it seems to be a mix.

Comment (putvande) [reply brands/paultons/src/less/type.less]: Time for some CSS linting?

Comment (nickine) [reply brands/paultons/src/less/type.less]: Oh this is where it started!

Comment (putvande) [reply brands/paultons/src/less/type.less]: Yes.

Comment (jlsorak) [reply brands/paultons/src/less/type.less]: Nope it's not just those boxes, it's for all links across the site. Without an underline, it's not clear that these are links, so @jodiedoubleday suggested to add the `text-decoration` to the links.

**Comment by jlsorak** [comment]
Following a suggestion from @putvande, I've changed the selector from `p > a` to `p a` as the anchor may not always be the paragraph's immediate child.

**Pull request merged**